src/components/Modals/AnnouncementModal.vue

```diff
@@ -32,6 +41,15 @@ export default {
       type: Object,
     },
   },
+  data() {
+    return {
+      USER,
+      ROLE,
+    };
```

Comment:
Because this data is constant and we don't need it to be reactive, we can just instantiate it in a created() hook. Just in case you want it, an outline of that process is explained at the bottom of [this article](https://coderethinked.com/3-different-ways-to-access-constants-in-a-vue-template/ ). It's a small change but just in the spirit of best practices.